doc/template.html

```diff
@@ -3,6 +3,7 @@
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width">
+  <meta name="nodejs.org:node-version" content="__VERSION__">
```

Comment:
@tolmasky, would you mind explaining what this meta tag will be used for (if anything)?

Reply:
> The problem it solves is that there is currently no easy way to determine what version of the docs you are viewing without falling back regex hacks on the title or h1 element, which should probably not be relied on as a future-proof method, especially when localizing the docs. The URL itself also isn't reliable since these docs often live at vXX-latest. This meta tag provides a place you can reliably query the version of the node docs, whether as a script on the page (as in the other commit), or a program operating on the page (for example if you are running end-to-end tests with puppeteer), or even when visually comparing a diff.
> &mdash; https://github.com/nodejs/node/pull/34148#issuecomment-659046901

I wonder if this meta tag would be useful for @zeke's localization work.